src/components/select-field/BaseSelectField.js

```diff
@@ -381,34 +399,47 @@ class BaseSelectField extends React.Component<Props, State> {
             listboxProps['aria-multiselectable'] = true;
```

Comment:
Is `listboxProps` still used in this class? It looks like it was moved into SelectDropdownList.